airbyte-ci/connectors/pipelines/pipelines/dagger/actions/secrets.py

```diff
@@ -19,14 +19,31 @@
     from pipelines.airbyte_ci.connectors.context import ConnectorContext
 
 
-async def get_secrets_to_mask(ci_credentials_with_downloaded_secrets: Container) -> list[str]:
+# List of overrides for the secrets masking logic.
+# These keywords may have been marked as secrets, perhaps somewhat aggressively.
+# Masking them, however, is annoying and pointless.
+# This list should be extended (carefully) as needed.
+NOT_REALLY_SECRETS = [
+    "admin",
+    "airbyte",
+    "host",
+]
```

Comment:
nit: can this be a `set`?
```suggestion
NOT_REALLY_SECRETS = {
    "admin",
    "airbyte",
    "host",
}
```
Would it be convenient to store this in a separate file?

Reply:
> Would it be convenient to store this in a separate file?

I don't see the point, this list is always going to be very short.